trulens/nn/models/__init__.py

```diff
@@ -1,17 +1,179 @@
+            performs the softmax. This layer should have an `activation`
+            attribute. Only used when `replace_softmax` is true.
+
+        custom_objects:
+            _Optional, for use with Keras models only._ A dictionary of
+            custom objects used by the Keras model.
+
+        input_shape:
+            _Required for use with Pytorch models only._ Tuple specifying
+            the input shape (excluding the batch dimension) expected by the
+            model.
+        
+        input_dtype: torch.dtype
+            _Optional, for use with Pytorch models only._, The dtype of the input.
+
+        device:
+            _Optional, for use with Pytorch models only._ A string
+            specifying the device to run the model on.
+
+        input_tensors:
+            _Required for use with TensorFlow 1 graph models only._ A list
+            of tensors representing the input to the model graph.
+
+        output_tensors:
+            _Required for use with TensorFlow 1 graph models only._ A list
+            of tensors representing the output to the model graph.
+
+        internal_tensor_dict:
+            _Optional, for use with TensorFlow 1 graph models only._ A
+            dictionary mapping user-selected layer names to the internal
+            tensors in the model graph that the user would like to expose.
+            This is provided to give more human-readable names to the layers
+            if desired. Internal tensors can also be accessed via the name
+            given to them by tensorflow.
+
+        default_feed_dict:
+            _Optional, for use with TensorFlow 1 graph models only._ A
+            dictionary of default values to give to tensors in the model
+            graph.
+
+        session:
+            _Optional, for use with TensorFlow 1 graph models only._ A 
+            `tf.Session` object to run the model graph in. If `None`, a new
+            temporary session will be generated every time the model is run.
+
+        backend:
+            _Optional, for forcing a specific backend._ String values recognized
+            are pytorch, tensorflow/tf, keras, or tf.keras.
+    """
+    if backend is None:
+        backend = discern_backend(model)
+        tru_logger.info(
+            "Detected {} backend for {}.".format(backend, type(model)))
+    # get existing backend
+    B = get_backend()
+
+    if B is None or (backend is not 'unknown' and B.backend != backend):
+        tru_logger.info(
+            "Changing backend from {} to {}.".format(
+                None if B is None else B.backend, backend))
+        os.environ['TRULENS_BACKEND'] = backend
+        B = get_backend()
     else:
-        from trulens.nn.models.tensorflow_v1 import TensorflowModelWrapper as ModelWrapper
+        tru_logger.info("Using backend {}.".format(B.backend))
+    tru_logger.info(
+        "If this seems incorrect, you can force the correct backend by passing the `backend` parameter."
+    )
```

Comment:
Wanted to keep this at a more visible level just in case our automated selection doesn't capture some usecase, the user will want to see this message.